server.elm

```diff
@@ -655,30 +682,168 @@ div#modify-menu div.keyvalues > div.keyvalueadder:hover {
   opacity: 1;
 }
 div#modify-menu input {
-  padding: 4px;
+  padding: 6px;
+  width: 100%;
 }
 div#modify-menu #newTagName {
-  width: 50%;
-  font-size: 1em;  
+  font-size: 1.4em;  
+  background-color: rgb(0, 178, 179);
+  font-family: monospace;
+  padding: 4px;
+  flex: 1;
 }
 div#modify-menu .tagname-info {
-  color: #d4d4d4;
   display: inline-block;
-  width: 50%;
+  border-radius: 4px;
   white-space: nowrap;
   overflow: hidden;
   text-overflow: ellipsis;
+  background: transparent;
+  color: whitesmoke;
+  flex: 2;
 }
 div#modify-menu input[type=radio] {
   width: initial;
   font-size: 1em;
 }
+
+div.keyvalue > span > input {
+  border-radius: 4px;
+  margin: 2px;
+}
+
+div.tagname-summary {
+  display: flex;
+  background-color: rgb(0, 178, 179);
+  border-radius: 0.4em;
+  padding: 4px;
+}
+
 .inline-input {
   background: transparent;
   color: white;
   border: none;
 }
 
+/* dom selector css */
+div.dom-selector-style {
+  margin: 2%;
+  border-radius: 0.4em;
+  padding: 0.4rem;
+  background: rgba(0, 212, 159, 0.6);
+}
+
+div.childrenElem {
+  display: flex;
+  height: 80px;
+}
+
+div.mainElemName {
+  color: white;
+  font-family: monospace;
+}
+
+div.childrenSelectorName {
+    font-family: monospace;
+}
+      
+div.childrenSelector {
+  min-width: 50px;
+  position: relative;
+  flex: 1; 
+  text-align: center; 
+  overflow: hidden;
+  padding: 10px;
+  margin: 2px;
+  /* border: 1px solid whitesmoke; */
+  border-radius: 0.3rem;
+  background-color: rgba(0, 212, 159);
+  transition: all 0.5s;
+  text-decoration: none;
+  cursor: pointer;
+}
+
+div.childrenSelector:hover {
+   -webkit-filter: contrast(125%); /* Chrome, Opera */
+      -moz-filter: blur(125%);
+      -ms-filter: blur(125%);    
+          filter: blur(125%);
+}
+
+div.mainElem {
+  height: 90px;
+  padding-bottom: 6px;
+  position: relative;
+  text-align: center;
+  font-size: 1.6em;
+  color: turquoise;
+  margin-bottom: 0.5em;
+}
+      
+div.mainElemInfo {
+  font-size: 0.6em;
+  color: whitesmoke;
+}
+
+div.childrenSelectorInfo {
+  text-overflow: ellipsis;
+  overflow: hidden;
+  color: whitesmoke;
```

Comment:
My comment is redundant, but I see a lot of whitesmoke. Use a variable instead and define the color in :root with a meaningful name, this will make future updates easier.